sw/tools/generators/gen_aircraft.ml

```diff
@@ -397,9 +397,9 @@ let () =
 
     (* Get TARGET env, needed to build modules.h according to the target *)
     let t = try Printf.sprintf "TARGET=%s" (Sys.getenv "TARGET") with _ -> "" in
+    make_opt "radio_ac_h" "RADIO" "radio";
```

Comment:
Since the project is made at the "all_ac_h" and it uses `gcc -MM` to get the files it also needs the radio header file.